src/utils/auth0.test.js

```diff
@@ -45,20 +45,20 @@ describe('webAuthHandler', () => {
   });
 });
 
-describe('initWebAuth', () => {
-  it('returns an initialized web auth', () => {
-    const webAuth = initWebAuth();
-    expect(auth0.WebAuth).toHaveBeenCalledWith({
-      domain: AUTH0_DOMAIN,
-      clientID: AUTH0_CLIENT_ID,
-      redirectUri: AUTH0_CALLBACK_URL,
-      audience: 'http://minimal-demo-iam.localhost:8000', // 'https://' + AUTH0_DOMAIN + '/userinfo',
-      responseType: 'token id_token',
-      scope: 'openid profile',
-    });
-    expect(webAuth).toEqual(webAuthMock);
-  });
-});
+// describe('initWebAuth', () => {
```

Comment:
This test is way "too internal" it doesn't test any business logic or exposed API code. Consider deleting.